README.md

```diff
@@ -1 +1,148 @@
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖OOP school library <a name="about-project"></a>
+
+**OOP school library** In this project, I start building my school library app. In this initial step, I implement the classes to represent students and teachers.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/fr/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- OOP school library: decorate a class
+- OOP school library: set up associations
+- OOP school library: add basic UI
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 You can check the Live Demo here<a name="live-demo"></a>
+
+The live demo will be availabe soon
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Step 1: Clone this project using this git comand "git clone", (if you don't have git installed please visit
+  https://git-scm.com/downloads and install it).
+- Step 2: Once you have installed git you can clone this project using the command below:
+  ```
+  git clone https://github.com/BenMKT/OOP-school-library.git
+  ```
+- Step 3: Once you have cloned this repo navigate to the folder that contains
+  the repository.
+- Step 4: If you want to watch the html file open it with your favorite web browser(This project MUST be opened from VS Code for full functionality).
+- Step 5: If you want to edit this project open it with your favorite text editor.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running cloning this repo `https://github.com/BenMKT/OOP-school-library.git`.
+
+### Prerequisites
+
+There are no known prerequisites to run this project.
+If you encounter any issues trying to run it, please feel free to
+[open a ticket](https://github.com/BenMKT/OOP-school-library/issues) to let me know and i will work on it to the best
+of my abilities
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git clone https://github.com/BenMKT/OOP-school-library.git`
+
+or
+
+`git clone git@github.com:BenMKT/OOP-school-library.git` <-- _Only if you have [SSH keys configured](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on your GitHub account_.
+
+### Usage
+
+To run the project:
+
+- Method 1:
+
+```sh
+  cd PATH_TO_CLONE
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> 👤 **Benson Kiai**
+- GitHub: [@Benson](https://github.com/BenMKT)
+
```

Comment:
- [ ] [Optional] I would kindly suggest you to include all your social media handles you only have the GitHub handle you have not included the Twitter and linkedin handles as this will make a recruiter or other developer to contact you easily Please Kindly make sure you include all of your social media handle